### ISSUE TITLE
vulkan api crashing.

### ISSUE THREAD

**Issue by Tekkenlord-X** [devmiyax/yabause#861]
Hi i am a windows user of yabasanshiro, currently i am using version 1.8.0 and when i try to use the vulkan renderer yabasanshiro just crashes, i can use open gl and the fake and software api but no vulkan for me, this is an issue i have since yabasanshiro 1.5.0, is it just me or this is happening for everyone?

**Comment by ipotatoe** [comment]
yeah, it happens with me as well, but all the renderers make the graphics messed up, so its unplayable to me

**Comment by Tekkenlord-X** [comment]
yes, same here, what happened to this great emulator? at some point it was very good now at least on windows it is useless if you compare with SSF or at some extent with kronos, because kronos also have really big messed up things right now, is like if they don't really care anymore about the windows builds.

**Comment by ipotatoe** [comment]
yeah, ikr?, all SS emulators i found either:
dont work at all
works but crashes either after some part of the game or randomly
works but lags/stutters a lot so its horrible to play
i guess the only thing we can do is just use old versions
